prepare-wp-core-update.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+cd "${GITHUB_WORKSPACE}/${PATH_DIR}" || exit 1;
+
+cp "$GITHUB_ACTION_PATH/handle-wp-core-update.sh" "${RUNNER_TEMP}/handle-wp-core-update.sh"
+
+CORE_VERSION_COMPOSER_TO=$(composer config extra.wordpress-core) || CORE_VERSION_COMPOSER_TO=""
+
+cd "${GITHUB_WORKSPACE}/${FROM_DIR}" || exit 1;
+
+CORE_VERSION_COMPOSER_FROM=$(composer config extra.wordpress-core) || CORE_VERSION_COMPOSER_FROM=""
```

Comment:
Issue here during onboarding. That would never be set for the first time the extras.wordpress-core is being added to a composer.json.

Reply:
yes, but it should not stop the thing from running